examples/translator/proto/translator.proto

```diff
@@ -17,6 +17,7 @@
 syntax = "proto3";
 
 package oak.examples.translator;
+option go_package = "github.com/project-oak/oak/examples/translator/proto";
```

Comment:
Out of curiosity, what's the default package they end up with otherwise?

Reply:
It gets `package oak_examples_translator` (which doesn't comply with the [Go convention](https://golang.org/doc/effective_go.html#package-names) that the package name should be the last part of the path)